mappings/net/minecraft/util/shape/VoxelShapes.mapping

```diff
@@ -99,6 +99,13 @@ CLASS net/minecraft/class_259 net/minecraft/util/shape/VoxelShapes
 	METHOD method_20713 unionCoversFullCube (Lnet/minecraft/class_265;Lnet/minecraft/class_265;)Z
 		ARG 0 one
 		ARG 1 two
+	METHOD method_31943 cuboidUnchecked (DDDDDD)Lnet/minecraft/class_265;
+		ARG 0 xMin
+		ARG 2 yMin
+		ARG 4 zMin
+		ARG 6 xMax
+		ARG 8 ymax
+		ARG 10 zMax
```

Comment:
```suggestion
		ARG 0 minX
		ARG 2 minY
		ARG 4 minZ
		ARG 6 maxX
		ARG 8 maxY
		ARG 10 maxZ
```
This order follows the conventions (and is already used below in `VoxelShapes.BoxConsumer`).